### ISSUE TITLE
Any update

### ISSUE THREAD

**Issue by kr812** [Lomusire/unlimited-cloud-storage#8]
Hey brother any update it's work or not in feature pls do something i request you

**Comment by Lomusire** [comment]
Yes it somwhat works now still in beta and seems to not run in the background corectly due to manifest v3 restrictions (wich probably could be fixed but curently idk a soulution that dosen't require to grant permission to every website) After completing miner it will move on to gemmerge where it only credits you coins every 100 levels
